safety_shield/scripts/urdf_capsule_generation.py

```diff
@@ -0,0 +1,527 @@
+import os
+import math
+import re
+import warnings
+
+import numpy as np
+import matplotlib.pyplot as plt
+import trimesh
+import xml.etree.ElementTree as ET
+from sklearn.decomposition import PCA
+
+# make console output more readable
+np.set_printoptions(threshold=np.inf)
+np.set_printoptions(suppress=True)
+
+
+def export_PC(point_cloud, name):
+    """
+    debug function: export point cloud to ply file (can be imported to blender)
+    :param point_cloud: the point cloud to export
+    :param name: name of the file
+    :return: void
+    """
+    print("saving to file: " + name)
+    with open(name + '.ply', "w") as file:
+        # HEADER
+        file.write("ply\n")
+        file.write("format ascii 1.0\n")
+        file.write("element vertex {}\n".format(point_cloud.shape[0]))
+        file.write("property float x\n")
+        file.write("property float y\n")
+        file.write("property float z\n")
+        file.write("end_header\n")
+
+        # write points
+        for p in point_cloud:
+            file.write(
+                "{} {} {} \n".format(p[0], p[1], p[2]))
+
+
+def visualize_meshes(*meshes):
```

Comment:
Why `*meshes` instead of `meshes` or `meshes = []`?

Reply:
Changed in ba77b39d61fff8cae93f6d27702cc05d1c493452 to make it more user-friendly